.github/workflows/e2e-tests.yml

```
@@ -127,35 +133,24 @@ jobs:
               body,
             });
 
-      - name: Log failure to failure-log.md
+      - name: Log failure to the job summary
         if: steps.results.outputs.status == 'red'
         run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)